components/molecules/ModalElement.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
```

Comment:
For the modal focus, I highly suggest checking out [focus-trap-react](https://www.npmjs.com/package/focus-trap-react)

I used it before in Benefits Finder [here](https://github.com/DTS-STN/Benefits-Finder/blob/main/components/organisms/Drawer.js)

The problem is that when the modal is open, we can still tab through the stuff in the back.

Reply:
Good catch!